### DISCUSSION TITLE
请问更改快速启动快捷键的办法

### DISCUSSION THREAD

**Discussion by piliwu** [MuiseDestiny/zotero-gpt#141] [Q&A]
原来是shift+/，现在加了一个shift+?。
但是我有打问号的习惯，就导致每次输入完问题一输入“？”后，输入的问题全没有了，相当于重新唤醒了。
想把shift+?这个快捷键去掉，只保留shift+/
谢谢

**Comment by MuiseDestiny** [comment]
ctrl ➕ /

写了❓避免按错，意思是既有斜杠也有问号的那个斜杠。

下个版本可能要去除shift触发。

**Reply by piliwu** [reply]
啊不好意思 我发现我描述的有问题
就是按shift+/会重新唤醒，然而我每次打“？”的时候都用shift+/，结果就每次都重新唤醒，问题白输入了
刚才试了用ctrl+/唤醒，但是依然每次一打“？”就又重新唤醒了
那我就先等等 等下一版更新去掉shift触发
谢谢啦
